workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationService.ts

```diff
@@ -181,11 +186,16 @@ export class DynamicAuthorizationService {
    * @param removeUserFromGroupRequest - {@link RemoveUserFromGroupRequest}
    *
    * @returns - {@link RemoveUserFromGroupResponse}
+   *
+   * @throws {@link IdpUnavailableError} - IdP encounters an error
+   * @throws {@link PluginConfigurationError} - plugin has a configuration error
+   * @throws {@link UserNotFoundError} - user could not be found
+   * @throws {@link GroupNotFoundError} - group could not be found
    */
   public async removeUserFromGroup(
     removeUserFromGroupRequest: RemoveUserFromGroupRequest
   ): Promise<RemoveUserFromGroupResponse> {
-    throw new Error('Not implemented');
+    return this._groupManagementPlugin.removeUserFromGroup(removeUserFromGroupRequest);
```

Comment:
add auditing to this

Reply:
Since auditing will be added to all routes, does it make sense to handling auditing everything in a separate ticket?

Reply:
auditing is not guaranteed to be added on all routes, we will need to audit all API endpoints that modify any resources.

Reply:
ok, so a separate ticket for auditing routes that require it? Auditing infrastructure isnt in develop so I cant add it currently.

Reply:
Yep, auditing infra will go in with #725.